### ISSUE TITLE
Reading M4B fails but same file as M4A works

### ISSUE THREAD

**Issue by ElmoFuntz** [Zeugma440/atldotnet#43]
## The problem

I was super excited to find this as I have been working on an audiobook chapterizer now that all the decent ones died with the death of quicktime but I have found that ATL appears to be unable to read M4B file data even though it is just an M4A.

## Environment

* ATL version: 2.15
* Last ATL version: NA
* OS/version used to run ATL: Win 10, VS 2018 .net 4.8

## Details

I am using your sample code for reading metadata which works (in most cases but filing a 2nd bug for a few outliers) when the file extension is .m4a

```
Title : Test file
Artist : 
Album : Test author
Recording year : 2019
Track number : 0
Disc number : 0
Genre : Audiobook
Comment : this is a comment
Duration (s) : 28504
Bitrate (KBps) : 68
Number of channels : 2
Channels arrangement : Stereo (2/0.0)
Has variable bitrate audio : yes
Has lossless audio : no
Custom field ©too : value = Lavf58.17.101
```
but fails with m4b extension (it is really an m4b file I renamed back to m4a)

```
d:/test.m4b | Instancing a Dummy Audio Data Reader for d:/test.m4b [?=5476 ms]
d:/test.m4b | Instancing a Dummy Meta Data Reader [?=1 ms]
d:/test.m4b | Could not find any metadata [?=0 ms]
Title : test
```

## Code To Reproduce Issue [ Good To Have ]

I am literally copying and using your sample read meta data code to test the ability of the library to read data I need.

**Comment by Zeugma440** [comment]
Hi and thanks for the feedback !

To be perfectly honest, I did not know about the existence of M4B files until today 😅 

I'll add that format to ATL (with the same internals as M4A) once I am back from vacations, i.e. beginning of January.

**Comment by Zeugma440** [comment]
Just added support for M4B files.

Can you pull the latest commit from `master` and tell me if it works for you ?

**Comment by ElmoFuntz** [comment]
Um sure.. soon as I figure out how :) I installed using nuget so pardon me while I go google how to do this manually..

**Comment by Zeugma440** [comment]
If you're not familiar with this, I'd better provide you with a temporary DLL from my side. Wait a sec...

**Comment by Zeugma440** [comment]
[ATL.zip](https://github.com/Zeugma440/atldotnet/files/4014525/ATL.zip)

**Comment by ElmoFuntz** [comment]
Awesome thanks. I probably really should learn this since i find myself wanting to look at projects on github more and more often.

**Comment by Zeugma440** [comment]
...does the fix work for you on your other files as well ? :D

**Comment by ElmoFuntz** [comment]
Ok so something weird happened with my original test. Somehow the old version 2.15 ended up getting copied back over. I replaced it with your copy updated my references and everything is looking great now. I tested on all of my original test files plus a few new ones one of which is 43 hours long and they all look good so far. I have not gotten too much into the chapter stuff yet and never got to it before the bug report but hopefully those will all be good as well.
